scrnaseq_hto.Rmd

```diff
@@ -380,40 +392,39 @@ ncounts_metrics = grep("_HTO",
                         invert=TRUE)
 
 p1 = VlnPlot(sc, features=ncounts_metrics, idents=levels(Seurat::Idents(sc)), pt.size=0) + 
-  geom_violin(color=NA) +
+  geom_violin(color=NA) + 
   AddStyle(title="Counts (RNA)", 
            fill=hto_colours$col_hto_collapsed, 
-           legend_title="Classified cells",
-           legend_position="bottom", 
-           xlab="")
+           legend_position="none", 
+           xlab="") +
+  theme(axis.text.x=element_text(angle=45, hjust=1, vjust=1)) 
 
 p2 = VlnPlot(sc, features=nfeature_metrics, idents=levels(Seurat::Idents(sc)), pt.size=0) + 
   geom_violin(color=NA) +
   AddStyle(title="Features (RNA)", 
            fill=hto_colours$col_hto_collapsed, 
-           legend_title="Classified cells",
-           legend_position="bottom", 
-           xlab="")
+           legend_position="none", 
+           xlab="") +
+  theme(axis.text.x=element_text(angle=45, hjust=1, vjust=1)) 
 
-p = patchwork::wrap_plots(list(p1, p2), ncol=2) & theme(legend.position="bottom") 
-p = p + patchwork::plot_annotation("Number of counts and features for HTO-classified cells") + 
-  patchwork::plot_layout(guides = "collect")
+p = patchwork::wrap_plots(list(p1, p2), ncol=2)
+p = p + patchwork::plot_annotation("Number of counts and features for HTO-classified cells")
 p
 ```
 
-# Remove cells classified as doublet or negative
+# Filtering singlets
 This section of the report states the number of cells that remain after negative and doublet cells are removed. 
 ```{r hto_subset}
 sc.all = sc
 sc = subset(sc, idents=c("Negative", "Doublet"), invert=TRUE)
 sc
 ```
 
-# Preliminary visualisation of demultiplexed RNA data
+# Preliminary visualisation of RNA data
 This section of the report provides first insights into your RNA dataset based on a preliminary pre-processing of the RNA data using the standard scRNA-seq workflow. 
-```{r prelim_analysis, results='hide'}
+```{r prelim_analysis, results="hide"}
 # Normalise, scale, pca umap and clustering for sc
-sc = Seurat::NormalizeData(sc, assay="RNA", normalization.method = "LogNormalize", scale.factor=10000, verbose=FALSE)
+sc = suppressWarnings(Seurat::NormalizeData(sc, assay="RNA", normalization.method = "LogNormalize", scale.factor=10000, verbose=FALSE))
```

Comment:
I found out that this is because of this: `future.plan="multicore",`. For now this is okay, but let us check the other future/parallelisation options before release.

Reply:
Created issue